dev_ops/sql/Tables/CreateContractDonations.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE contract_donations (
```

Comment:
I really like wrapping all sql queries in a BEGIN COMMIT block. I think its a good pattern to follow, but im curious to hear your thoughts on this.

Begin;

.
.
.

COMMIT;

Reply:
A BEGIN COMMIT wrapper is specifying a sql transaction block. Since this is just a simple 1 line sql for setup it isn't really need. This pattern can be followed for sql queries that aren't dev setup

Reply:
It's not necessary for single DDL statements like this because they are already transactions. But it's probably a good idea to add this to the CreateSchema.sql file so creating the schema is atomic.